apps/main-landing/src/app/donors/[uuid]/page.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+import { useWallet, WalletContextProvider } from '@idriss-xyz/wallet-connect';
+import NiceModal from '@ebay/nice-modal-react';
+import { Button } from '@idriss-xyz/ui/button';
+import { WithPortal } from '@idriss-xyz/ui/providers/with-portal';
+
+// ts-unused-exports:disable-next-line
+export default function Donors() {
+  return (
+    <WithPortal>
+      <NiceModal.Provider>
+        <WalletContextProvider>
+          <Foo />
+        </WalletContextProvider>
+      </NiceModal.Provider>
+    </WithPortal>
+  );
+}
+
+const Foo = () => {
```

Comment:
are we staying with `Foo` as a name?

Reply:
haha no :D it's just for showcase of Wallet connect modal in donors page for tomorrow